plasmapy/diagnostics/proton_radiography.py

```diff
@@ -300,6 +305,212 @@ def _log(self, msg):
     # Define some constants so they don't get constantly re-evaluated
     _c = const.c.si.value
 
+    # *************************************************************************
+    # Create mesh
+    # *************************************************************************
+
+    def add_wire_mesh(
+        self, location, extent, nwires, wire_diameter, mesh_hdir=None, mesh_vdir=None
+    ):
+        """
+        Add a wire mesh grid between the particle source and the object grid
+        that blocks particles which intersect the wires.
```

Comment:
```suggestion
        that blocks particles whose paths intersect the wires.
```
Unless those are pretty chunky particles.